src/entry.rs

```diff
@@ -5,7 +5,7 @@ use regex::Regex;
 
 fn type_translation(input: String) -> String {
```

Comment:
I think we might as well change this function so we can completely remove the calls to `clone`.
```suggestion
fn type_translation(input: &str) -> String {
```